clean-abap/CleanABAP_es.md

```diff
@@ -0,0 +1,5127 @@
+
+Por este motivo, por favor toma en cuenta que no introducimos todos los conceptos en el mismo detalle
+que el libro original y las fuentes mencionadas: estas fuentes aún vale la pena que sean leídas, especialmente
+si no estás de acuerdo con elementos de esta guía porque no los explicamos tan bien.
+Usa las ligas en las secciones para leer el trasfondo de nuestra guía.
+
+Eres libre de discutir y estar en desacuerdo con cualquier cosa que decimos aquí.
+Uno de los pilares de Clean Code es que _el equipo manda_. Solo asegúrate de 
+darle a cada cosa una oportunidad justa antes de descartarla.
+
+[CONTRIBUTING.md](../CONTRIBUTING.md) sugiere maneras en que puedes cambiar esta guía o desviarte de ella en detalles menores. Toda contribución o discusión se lleva a cabo en inglés.
+
+## Nomenclatura
+
+> [Clean ABAP](#clean-abap) > [Contenido](#contenido) > [Esta sección](#nomenclatura)
+
+### Usa nombres descriptivos
+
+> [Clean ABAP](#clean-abap) > [Contenido](#contenido) > [Nomenclatura](#nomenclatura) > [Esta sección](#use-descriptive-names)
+
+Usa nombres que muestren el contenido y significado de las cosas.
+
+```ABAP
+CONSTANTS max_wait_time_in_seconds TYPE i ...
+DATA customizing_entries TYPE STANDARD TABLE ...
+METHODS read_user_preferences ...
+CLASS /clean/user_preference_reader ...
+```
+
+No te enfoques en el tipo de datos o codificación técnica.
+Rara vez contribuyen a entender el código.
+
+```ABAP
+" anti-pattern
+CONSTANTS sysubrc_04 TYPE sysubrc ...
+DATA iso3166tab TYPE STANDARD TABLE ...
+METHODS read_t005 ...
+CLASS /dirty/t005_reader ...
+```
+
+[No trates de arreglar nombres malos con comentarios.](#los-comentarios-no-son-excusa-para-nombrar-mal-objetos)
+
+> Lee más en el _Capítulo 2: Nombres con sentido: Usar nombres que revelen las intencioness_ de [_Código Limpio_ por Robert C. Martin].
+
+### Prefiere nombres del dominio de solución o de problema
+
+> [Clean ABAP](#clean-abap) > [Contenido](#contenido) > [Nomenclatura](#nomenclatura) > [Esta sección](#prefiere-nombres-del-dominio-de-solución-o-de-problema)
+
+Busca nombres buenos en el dominio de solución, por ejemplo en ciencias de la computación términos como
+"pila" o "árbol"; y en el dominio de problema, por ejemplo "cuenta" o "libro_mayor".
+
+Las capas que pertenecen a negocio cuando son nombradas de acuerdo al dominio de problema.
+Esto especialmente cierto para componetes que son diseñados bajo el diseño guiado por el dominio, como las APIs
+y los objetos de negocio.
+
+Las capas que proveen la mayor parte de la funcionalidad técnica, como las clases de fábrica y algoritmos abstractos, 
+sonarán mejor cuando son nombradas de acuerdo al dominio de solución.
+
+En cualquier caso, no trates de hacer tu propio lenguaje. Necesitamos poder intercambiar información
+entre desarrolladores, product owners y clientes, así que elige nombres que todos entiendan
+sin requerir un diccionario a la medida.
```

Comment:
Removing that word in this case looks and sounds unnatural, that might be due to differences between Latin American Spanish and other variants. "a la medida" is usually the Spanish way of saying custom.

Reply:
Might be, I am from Spain, so my Spanish is certainly different in some regards. Here we say "a medida" when something is custom made.

In that case use the phrasing you wish, since both are apparently correct.

Reply:
I did some quick research and you're right, both seem to be correct [as shown here](https://dle.rae.es/medida).

I decided to keep it as is based on Google search results by exact phrase, "a la medida" returns 126M results, while "a medida" returns 123M.